src/server_manager/scripts/fill_packaging_opts.sh

```diff
@@ -22,7 +22,9 @@
 # Note that you MUST use "source" in order to run the script in the same process as the calling
 # script, allowing fill_packaging_opts.sh to fill variables for the caller.
 
-readonly YARN_COMMAND="yarn do $(echo "${1}" | sed 's:.*\(src/server_manager/electron_app/.*\)_action.sh:\1:')"
+readonly ELECTRON_PATH='server_manager/electron_app/'
+readonly RELATIVE="${1#*/src/${ELECTRON_PATH}}"
```

Comment:
This was very confusing. It's relative to src/, but it felt like it was relative to ELECTRON_PATH.
Perhaps add a comment?

Reply:
BTW, that was wrong with the original code? That seemed more readable.

Reply:
Shellcheck warns about using `sed` here.  (The style guide [agrees](https://google.github.io/styleguide/shellguide.html#builtin-commands-vs-external-commands).)  Instead, it recommends using Bash's built-in substitution capabilities.

BTW, the original `sed` invocation is wrong: it produces output like `yarn do src/server_manager/...` instead of `yarn do server_manager/...`.